source/NoteRefFilter.cpp

```diff
@@ -4,6 +4,7 @@
 // System libraries
 #include <NodeInfo.h>
 
+#include <string.h>
```

Comment:
Are you sure it's needed? Compiles without it for me on 32bit.

Reply:
Oh, it doesn't work on 64-bit. This is what g++ says when I compile it on 64-bit: 
`~/TakeNotes/source> make
mkdir -p objects.x86_64-cc7-release; \
mkdepend  -I./  -Iobjects.x86_64-cc7-release/ -p .cpp:objects.x86_64-cc7-release/%n.o -m -f "objects.x86_64-cc7-release/NoteRefFilter.d" NoteRefFilter.cpp
g++ -c NoteRefFilter.cpp -iquote./  -iquote./  -iquoteobjects.x86_64-cc7-release/   -O3    -o "objects.x86_64-cc7-release/NoteRefFilter.o"
NoteRefFilter.cpp: In member function 'virtual bool NoteRefFilter::Filter(const entry_ref*, BNode*, stat_beos*, const char*)':
NoteRefFilter.cpp:39:6: error: 'strcmp' was not declared in this scope
  if (strcmp(mimeType, "application/takenotes") == 0)
      ^~~~~~
NoteRefFilter.cpp:39:6: note: suggested alternative: 'strtol'
  if (strcmp(mimeType, "application/takenotes") == 0)
      ^~~~~~
      strtol
/boot/system/develop/etc/makefile-engine:291: recipe for target 'objects.x86_64-cc7-release/NoteRefFilter.o' failed
make: *** [objects.x86_64-cc7-release/NoteRefFilter.o] Error 1`

`strcmp` is apparently a function to compare C-syle strings, which needs the `<string.h>` header.